test/shipping_zone_test.js

```diff
@@ -0,0 +1,28 @@
+var helper = require("./common.js");
+helper.setObject("shipping_zone");
+
+var Resource = helper.resource();
+
+helper.nock(helper.test_shop)
+  .get('/admin/shipping_zones.json')
+  .reply(200, helper.load("all"), {
+    server: 'nginx',
+    status: '200 OK'
+  });
+
+describe('Shipping zone', function() {
+  var site = helper.endpoint;
+  var resource = new Resource(site);
+
+  it('should get all shipping zones', function(done) {
+    resource.all(function(err, res){
+      res.should.not.be.empty;
```

Comment:
Use the previously saved fixture, e.g.`res.should.be.eql(fixture.shipping_zones)`;